features/search-input-type.yml

```diff
@@ -5,3 +5,4 @@ group: forms
 caniuse: input-search
 # TODO: investigate minor differences in the early support of the feature
 # between https://caniuse.com/input-search and our generated status.
+use_ranges_before_baseline_low_date: true
```

Comment:
I think that would provide a useful clue to someone reading the source and wondering if they can use the data through a published package. However, it wouldn't make it any easier for us to filter out such keys, because in order to catch typos we have to enumerate all allowed keys.